sdk/scheduler/src/main/java/com/mesosphere/sdk/config/DefaultConfigurationUpdater.java

```diff
@@ -30,6 +28,7 @@
 public class DefaultConfigurationUpdater implements ConfigurationUpdater<ServiceSpec> {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(DefaultConfigurationUpdater.class);
+    private static final String USER = "root";
```

Comment:
It seems like changing the variable itself to `DEFAULT_USER` makes more sense as setting a user at the scheduler level will propagate the user down to tasks thereby setting the user for the whole service.

In that case, moving it to [DcosConstants](https://github.com/mesosphere/dcos-commons/blob/master/sdk/scheduler/src/main/java/com/mesosphere/sdk/dcos/DcosConstants.java) seems reasonable as the class generically "encapsulates constants common to DC/OS and its services."